README.md

```diff
@@ -10,22 +10,22 @@ A list of popular sites and whether or not they accept two factor auth.
 
 ## The Goal
 
-The goal is to build a website ([TwoFactorAuth.org](https://twofactorauth.org)) with a list of popular sites that support
+The goal of this project is to build a website ([TwoFactorAuth.org](https://twofactorauth.org)) with a list of popular sites that support
 Two Factor Authentication, as well as the methods that they provide.
 
 Our hope is to aid consumers who are deciding between alternative services based on the security they
-offer for their customers. This can also serve as an indicator for the effort a site has put into security in general.
+offer for their customers. This project also serves as an indicator of general security efforts used on a site. 
 
 ## Contributing
 
-If you'd like to contribute, read the entire guidelines here in
+If you would like to contribute, please read the entire guidelines here in
 [CONTRIBUTING.md][contrib].
 
 ## Running Locally
 
 TwoFactorAuth.org is built upon [Jekyll](https://jekyllrb.com/), using the [github-pages](https://github.com/github/pages-gem) gem.
-In order to run the site locally, it is necessary to install bundler, install all dependencies, and then use Jekyll to serve
-the site. If the `gem` command is not available to you, it is necessary to install Ruby with RubyGems.
+In order to run the site locally, bundler, and all other dependencies will need to be installed, and afterwards Jekyll can be serve
```

Comment:
That should either be "Jekyll can serve" or "Jekyll can be used to serve".

Reply:
oops. I have fixed it :)